### ISSUE TITLE
Getting FileNotFound errors when running example tests

### ISSUE THREAD

**Issue by KaneCoulton** [hitchdev/hitchstory#10]
Hi, 
I've recently discovered hitchstory and been trying to get the rewrite test_correct_my_spelling example working.
The issue I'm currently seeing is that when running `$ STORYMODE=rewrite ./run.sh pytest -k test_correct_my_spelling` I'm getting the error `No such file or directory: '/src/tests/docstory.yml`.

I'm wondering if this is to do with the recent directory changes?

It appears a lot of the links on that example page are also broken as a result of the directoy changes.

Or is there something I'm doing wrong?

Thanks

**Comment by crdoconnor** [comment]
yes, it almost certainly is some changes I made recently to the examples.
thank you for bringing it up I will try to fix it a bit later on today.

**Comment by crdoconnor** [comment]
which example were you using - website, command line or rest api?

**Comment by KaneCoulton** [comment]
website
